src/components/Common/UI/Alert/Alert.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useId, useRef } from 'react'
+import { type AlertProps } from './AlertTypes'
+import styles from './Alert.module.scss'
+import InfoIcon from '@/assets/icons/info.svg?react'
+import SuccessIcon from '@/assets/icons/success_check.svg?react'
+import WarningIcon from '@/assets/icons/warning.svg?react'
+import ErrorIcon from '@/assets/icons/error.svg?react'
+
+export function Alert({ label, children, status = 'info', icon }: AlertProps) {
+  const id = useId()
+  const alertRef = useRef<HTMLDivElement>(null)
+  const defaultIcon =
+    status === 'info' ? (
+      <InfoIcon aria-hidden />
+    ) : status === 'success' ? (
+      <SuccessIcon aria-hidden />
+    ) : status === 'warning' ? (
+      <WarningIcon aria-hidden />
+    ) : (
+      <ErrorIcon aria-hidden />
+    )
+
+  useEffect(() => {
+    if (alertRef.current) alertRef.current.scrollIntoView({ behavior: 'smooth' })
+  }, [])
+
+  return (
+    <div className={styles.root}>
+      <div
+        className={styles.alert}
+        role="alert"
+        aria-labelledby={id}
+        data-variant={status}
+        ref={alertRef}
+      >
+        <div className={styles.icon}>{icon || defaultIcon}</div>
+        <h6 id={id}>{label}</h6>
```

Comment:
Nit: do we need to allow for setting a heading level as part of this? ex. https://github.com/Gusto/embedded-react-sdk/blob/04a7d21a2e0f03ee711827f84e4023eba55fb9d7/src/components/Common/DocumentViewer/DocumentViewer.tsx#L12

Was wondering from an a11y perspective how we can keep the heading levels following a logical order when this could be anywhere on the page

Reply:
Hmmm this is a good question, I'm less worried about going from h2 to h4.... but this is a good thing we should think about perhaps in the audit? We might be able to get a feel for what it should be when evaluating the whole page.